src/groundStation/ftp.py

```diff
@@ -0,0 +1,266 @@
+from groundStation.groundStation import groundStation, options
+from groundStation.system import SystemValues
+import libcsp_py3 as libcsp
+from random import randint
+import os
+
+class ftp(groundStation):
+    def __init__(self, opts):
+        self.operation = ''
+        self.infile = ''
+        self.outfile = ''
+        if opts.get and opts.post:
+            raise ValueError("Cannot post and get a file at the same time")
+        if not opts.get and not opts.post:
+            raise ValueError("Must get or post a file")
+        
+        if opts.get:
+            self.operation = "get"
+        elif opts.post:
+            self.operation = "post"
+
+        if self.operation == "get":
+            self.infile = opts.get
+        elif self.operation == "post":
+            self.infile = opts.post
```

Comment:
```suggestion
        if opts.get:
            self.operation = "get"
            self.infile = opts.get
        elif opts.post:
            self.operation = "post"
            self.infile = opts.post
```
It looks like we can condense these two if statements